### PR TITLE
#156 add frontend documentation

### PR DESCRIPTION
**Pull request by bejow**
## Related Ticket/Issue
#156 

## Description
An updated version of our README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,10 +20,43 @@ Feel free to contribute.
 
 ## Precommit- / Prepush- hook
 
--   This Project runs all test and applyies all linter rules before commit or push
+-   This Project runs all tests and applyies all linter rules before commit or push
```
Comment (ludwigschubi) [inline README.md]: *applies

File: README.md [hunk 2]
```diff
@@ -15,10 +20,43 @@ Feel free to contribute.
...
+
+### Workflow
+
+-   Select a Ticket from the [Project board](https://github.com/orgs/0wntech/projects/1)
+-   Check all the Ticket requirements
+-   Do your magic
+-   Write Tests for it (we can only accept tested code)
+-   Create a Pull request
+-   Let someone review it
+-   Congratulation, if your Pull Request gets accepted you contributed to the private web v1.0
```
Comment (ludwigschubi) [inline README.md]: *Congratulations

File: README.md [hunk 3]
```diff
@@ -15,10 +20,43 @@ Feel free to contribute.
...
+This is an OpenSource Project, be part of it!
+
+### Workflow
+
+-   Select a Ticket from the [Project board](https://github.com/orgs/0wntech/projects/1)
+-   Check all the Ticket requirements
+-   Do your magic
+-   Write Tests for it (we can only accept tested code)
+-   Create a Pull request
+-   Let someone review it
```
Comment (ludwigschubi) [inline README.md]: Wait for somebody to review your changes is better imo

File: README.md [hunk 4]
```diff
@@ -15,10 +20,43 @@ Feel free to contribute.
...
 
--   This Project runs all test and applyies all linter rules before commit or push
+-   This Project runs all tests and applyies all linter rules before commit or push
 -   If you have linter errors try `eslint "**/*.js` in terminal or remove the linter errors by hand
 
 ## Tests and Coverage
 
 -   Tests are written with the Jest framework
 -   Travis CI + codecov for testing and coverage report (report at [codecov](https://codecov.io/gh/0wntech/drive))
+-   To test in a live environment push your branch to [staging](https://github.com/0wntech/drive/tree/staging) it will deployed at [staging.drive.owntech](https://staging.drive.owntech.io/)
```
Comment (ludwigschubi) [inline README.md]: Can really anybody deploy to staging?

Comment (bejow) [reply README.md]: Yes, I forgot to restrict that, good catch! I now changed it that only certain people are allowed to push to staging. We can set them in the settings section of github.

**Comment by bejow** [comment]
:+1:  I corrected the spellings and restricted the staging branch for external people. @ludwigschubi please review again

**Pull request merged**